ecs-cli/modules/commands/compose/compose_command.go

```diff
@@ -79,10 +80,10 @@ func composeFlags() []cli.Flag {
 			Name:  command.VerboseFlag + ",debug",
 			Usage: "Increase the verbosity of command output to aid in diagnostics.",
 		},
-		cli.StringFlag{
+		cli.StringSliceFlag{
 			Name:   command.ComposeFileNameFlag + ",f",
-			Usage:  "Specifies the Docker compose file to use. Defaults to " + composeFileNameDefaultValue + " file.",
-			Value:  composeFileNameDefaultValue,
+			Usage:  "Specifies one or more Docker compose files to use. Defaults to " + composeFileNameDefaultValue + " file, and an optional " + composeOverrideFileNameDefaultValue + " file.",
+			Value:  &cli.StringSlice{},
```

Comment:
Nit:
The CLI's help prints this default in the format:
Name         Usage (default: "Value") [EnvVar]

Instead of an empty StringSlice, we should set Value to [composeFileNameDefaultValue, composeOverrideFileNameDefaultValue]